lib/curl.ml

```diff
@@ -60,6 +60,8 @@ let upload_archive ~archive ~user ~repo ~release_id =
       "-S";
       "-K";
       "-";
+      "-D";
+      "-";
```

Comment:
For me it has been a bit hard to understand why the "-D -" is necessary even after reading the commit message. I've understood it after reading the discussion. I'm wondering if it might be worth improving the commit message (or even leaving a comment here); for example, by mentioning that having the headers in the body messes up the JSON-structure of the body and by choosing a sentence that makes clear the causality (no "-D" argument => headers in the body). The latter e.g. by replacing "missing" with "added" in your commit message or so. What do you think?

Reply:
That's true this is a bit obscure. I'm not sure the commit message is a good place for this, I was thinking of adding a few comments in the code to explain the arguments or define a simple abstraction to have a named variable for each argument, instead of handling string literals (`curly` should have done that, since it's only `curl` arguments but it can't hurt doing it ourselves) but this is a bigger change so it would be in another PR.

Reply:
I like the idea of making that precise inside the code either by named variables or by comments. So yes, let's leave that for another PR.